packages/ra-core/src/sideEffect/redirection.spec.ts

```diff
@@ -34,12 +33,4 @@ describe('redirection saga', () => {
         const generator = handleRedirection(action);
         expect(generator.next().value).toEqual(put(push('/posts/123')));
     });
-
-    it('should yield a form refresh if redirectTo is falsy', () => {
```

Comment:
How do we implement the "save and add" then?

Reply:
It works on the simple example ;)